enterprise/admin/utils.py

```diff
@@ -111,4 +111,18 @@ def validate_email_to_link(email, raw_email=None, message_template=None, ignore_
         raise ValidationError(ValidationMessages.USER_ALREADY_REGISTERED.format(
             email=email, ec_name=existing_record.enterprise_customer.name
         ))
-    return bool(existing_record)
+    return existing_record or False
```

Comment:
Okay, as long as we're actually using the returned information for some reason, I'm good with it.